doc/integrator/objectstorage.rst

```diff
@@ -177,6 +177,13 @@ Azure:
 
 `Some more information <https://github.com/mapserver/mapserver/wiki/Render-images-straight-out-of-S3-with-the-vsicurl-driver>`_
 
+.. note::
+
+   If you want to be able to use different Bucket or Container in different environment
+   (like integration / production), you should have a file names ``<name>.raster``
```

Comment:
I don't understand what these file names refer to.

Maybe an example would help.

Reply:
I try to clarify ...